src/Libplanet.Net/Swarm.BlockCandidate.cs

```diff
@@ -146,25 +146,25 @@ private BlockChain AppendPreviousBlocks(
             bool render,
             IProgress<BlockSyncState> progress)
         {
-            BlockChain workspace = blockChain;
+            BlockChain workspace;
             List<Guid> scope = new List<Guid>();
             bool forked = false;
 
-            Block oldTip = workspace.Tip;
+            Block oldTip = blockChain.Tip;
             Block newTip = candidate.Blocks.Last().Item1;
```

Comment:
Is this variable still needed?

Reply:
Well, reorg hasn't been completely removed as you see in the code below. The goal is to remove it eventually. 😓

Reply:
I could find reorg still exists, but seems like `FindBranchPoint()` does not receives `newTip` at all, and find it from `newBlocks` inside the method.

Reply:
Ah, I misunderstood. Indeed, `newTip` can be removed. 😅